number_parser/parser.py

```diff
@@ -1,144 +1,136 @@
 import re
-import string
-# The initial global dictionaries that have been declared below will be changed,
-# should be easily derived automatically from CLDR data/other source for each of the languages.
-
-# We will need to have all values from 1-100 in a dictionary for some languages like hindi while others
-# english / french build 2 digit numbers using MTEN dictionary values.
-# (Difference observable in hi.json and en.json in numeral_language_data)
-
-MULTIPLIERS = {
-    "thousand": 1000,
-    "thousands": 1000,
-    "million": 1000000,
-    "millions": 1000000,
-    "billion": 1000000000,
-    "billions": 1000000000,
-    "trillion": 1000000000000,
-    "trillions": 1000000000000,
-}
-
-VALID_TOKENS_IN_NUMBERS = ["and", "-"]
-
-UNITS = {
-    word: value
-    for value, word in enumerate(
-        "one two three four five six seven eight nine".split(), 1
-    )
-}
-
-STENS = {
-    word: value
-    for value, word in enumerate(
-        "ten eleven twelve thirteen fourteen fifteen sixteen seventeen eighteen nineteen".split(), 10
-    )
-}
-
-MTENS = {
-    word: value * 10
-    for value, word in enumerate(
-        "twenty thirty forty fifty sixty seventy eighty ninety".split(), 2
-    )
-}
-
-HUNDRED = {"hundred": 100, "hundreds": 100}
-
-ALL_WORDS = {**UNITS, **STENS, **MTENS, **HUNDRED, **MULTIPLIERS}
-BASE_NUMBERS = {**UNITS, **STENS}
-
-def check_validity(current_token, previous_token):
+from importlib import import_module
+SENTENCE_SEPERATORS = [".", ","]
+
+
+class Language:
```

Comment:
Good job creating this class!!

I think we should rename it to something like `LanguageData`, as it doesn't represent a "language" but a "language corpus" (or "language data").

After this change, in the other places where you uses the `lang` variable name I would rename it to something in accordance like `lang_data`.